RecoEgamma/EgammaIsolationAlgos/python/egmPhotonIsolationAOD_cff.py

```diff
@@ -0,0 +1,40 @@
+import FWCore.ParameterSet.Config as cms
+
+from CommonTools.ParticleFlow.pfNoPileUpIso_cff import * 
+from CommonTools.ParticleFlow.pfParticleSelection_cff import * 
+
+
+pfNoPileUpCandidates = pfAllChargedHadrons.clone()
+pfNoPileUpCandidates.pdgId.extend(pfAllNeutralHadronsAndPhotons.pdgId)
+
+particleFlowTmpPtrs = cms.EDProducer("PFCandidateFwdPtrProducer",
+src = cms.InputTag('particleFlow')
+)
+
+egmPhotonIsolationAOD = cms.EDProducer( "CITKPFIsolationSumProducer",
+			  srcToIsolate = cms.InputTag("gedPhotons"),
+			  srcForIsolationCone = cms.InputTag('pfNoPileUpCandidates'),
+			  isolationConeDefinitions = cms.VPSet(
```

Comment:
egmPhotonIsolationAOD_cff.egmPhotonIsolationAOD
egmPhotonIsolationMiniAOD_cff.egmPhotonIsolationMiniAOD and 
egmPhotonIsolationPUPPI_cff.egmPhotonIsolationAODPUPPI 
all have the same content of isolationConeDefinitions
If it is the same, there should be no copy-paste of ~20 lines of values, but only one definition which is then loaded everywhere it is needed